docs/core/tools/project-json.md

```diff
@@ -1479,33 +1609,50 @@ For example:
 ### bin
 Type: Object
 
-An object with a single property, `assembly`, whose value is the assembly path.
+This is used wrap a DLL file. You can reference and generate a package containing this DLL. 
+
+It contains a single String  property, `assembly`, whose value is the assembly path.   
 
 For example:
 
 ```json
 "frameworks": {
     "netcoreapp1.0": {
         "bin": {
-            "assembly" :"c:/otherProject/otherdll.dll"
+            "assembly": "c:/otherProject/otherdll.dll"
         }
     }
 }
 ```
 
-### imports
-Type: String
+## runtimes
+Type: Object
 
-Specifies other framework profiles that this project is compatible with.
+List of [runtime identifiers (RIDs)](../rid-catalog.md) supported by the project (used when publishing [self-contained deployments](../deploying/index.md#self-contained-deployments-scd)).
 
 For example:
 
 ```json
-"frameworks": {
-    "netcoreapp1.0": {
-        "imports": "portable-net45+win8"
-    }
+"runtimes": {
```

Comment:
Couldn't this be any of the list of RIDs? That makes it a long and changing list, so it might be wise to point to that document.